vulnerabilities/importers/elixir_security.py

```diff
@@ -74,23 +70,18 @@ def collect_packages(self):
         packages = set()
         files = self._updated_files.union(self._added_files)
         for f in files:
-            with open(f) as file:
-                data = yaml.safe_load(file)
-                if data.get("package"):
-                    packages.add(data["package"])
+            data = load_yaml(f)
+            if data.get("package"):
+                packages.add(data["package"])
 
         return packages
 
-    def generate_all_version_list(self, pkg_name):
+    def get_versions_from_range(self, version_list, pkg_name):
+        pkg_versions = []
         if not getattr(self, 'pkg_manager_api', None):
             self.pkg_manager_api = HexVersionAPI()
-        version_list = self.pkg_manager_api.get(
+        all_version_list = self.pkg_manager_api.get(
             pkg_name)
-        return version_list
-
-    def get_versions_from_range(self, version_list, pkg_name):
-        pkg_versions = []
-        all_version_list = self.generate_all_version_list(pkg_name)
         if version_list is None:
             return
```

Comment:
returning an empty list would be better, for reasons

1. Eliminate lot of code you have to deal with this `None` return below.
2. Intuitively IMHO it makes more sense to me to return empty list when there are no versions. The function remains consistent that way.

Reply:
Ok